src/components/contentful/contentfulOpenLetterForm/writeLetterForm.js

```diff
@@ -31,7 +31,7 @@ const WriteLetterForm = ({ accessKey, accessPassword, language }) => {
       }
       axios
         .post(`${SERVICE_API_URL}/online-letter/send`, {
-          accessKey,
+          letterKey,
```

Comment:
I only found out after merging the PR that this is sending an invalid request body, since the API was still expecting `accessKey` :sweat_smile: So I decided to push a hotfix for it in master branch.

Reply:
Oh, now that I think of it, I tested it without actually sending a letter (so, just toggling the buttons), so thanks for catching & fixing!